scripts/install.sh

```diff
@@ -63,54 +66,52 @@ shift $(expr $OPTIND - 1) # remove options from positional parameters
 # Overwrite default properties
 CONFIG_DIR_PATH=${config_dir_path:-$CONFIG_DIR_PATH_DEFAULT}
 NETWORK=${network:-$NETWORK_DEFAULT}
-CONTRACTS_ONLY=${contracts_only:-false}
+SKIP_DEPLOYMENT=${skip_deployment:-false}
+SKIP_CLIENT_BUILD=${skip_client_build:-false}
 
 # Run script
 printf "${LOG_START}Starting installation...${LOG_END}"
 
 printf "Config dir path: $CONFIG_DIR_PATH\n"
```

Comment:
Do we use `$CONFIG_DIR_PATH` anywhere other than here? I can't see it being used anywhere.

Reply:
Right, it's not used anymore

Reply:
Then I think we can also delete:
https://github.com/keep-network/keep-core/blob/17d3a2cc12aa9bad352fc175ec553c4bbb697590/scripts/install.sh#L13,
https://github.com/keep-network/keep-core/blob/17d3a2cc12aa9bad352fc175ec553c4bbb697590/scripts/install.sh#L21,
https://github.com/keep-network/keep-core/blob/17d3a2cc12aa9bad352fc175ec553c4bbb697590/scripts/install.sh#L42,
https://github.com/keep-network/keep-core/blob/17d3a2cc12aa9bad352fc175ec553c4bbb697590/scripts/install.sh#L56
and modify https://github.com/keep-network/keep-core/blob/17d3a2cc12aa9bad352fc175ec553c4bbb697590/scripts/install.sh#L53.